ci-operator/config/che-incubator/che-test-harness/che-incubator-che-test-harness-master.yaml

```diff
@@ -27,7 +27,7 @@ resources:
       memory: 4Gi
 tests:
 - as: performance-tests
-  cron: 0 */12 * * *
+  cron: 0 */12 * * 0
```

Comment:
This would run twice each Sunday, right?
I think we should have:
`<random(60)>  <random in range low load hours> */14 * *`

Reply:
@jupierce I assume random will be determined in Bear's script and hardcoded into yaml, right?

Reply:
Right